routes/posts.test.js

```diff
@@ -0,0 +1,156 @@
+import request from 'supertest'
+import app from '../App.js'
+import { expect, test } from '@jest/globals'
+import { pool } from '../db/index.js'
+
+// Testing array of all the posts
+
+describe('GET api/posts', function () {
```

Comment:
This works well because we are populating the database with seed data every time the database starts up. We would not normally be doing this in normal scenarios, so it may be good to practice having a test hook for seeding the database and for clearing the database once testing has completed